pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -182,6 +184,8 @@ type cgroupManagerImpl struct {
 	subsystems *CgroupSubsystems
 	// simplifies interaction with libcontainer and its cgroup managers
 	adapter *libcontainerAdapter
+	// is the host running on cgroup v2
+	cgroupv2 bool
```

Comment:
nit: since this is already cached in https://github.com/opencontainers/runc/blob/master/libcontainer/cgroups/utils.go, can we just call IsCgroup2UnifiedMode each time we need it instead of caching the result here?

Reply:
good point, thanks for noticing it.

I've pushed a fixed version